### ISSUE TITLE
Removing or updating Gamescope brake Plasma6 wayland session on Manjaro 24.0. 

### ISSUE THREAD

**Issue by leucome** [ValveSoftware/gamescope#1312]
I am not so sure how to diagnose this and also how and why it could happen. Maybe somebody may have an idea how to deal with this or what log to look at or something. 

 When I upgraded Manjoro to 24.0 with plasma 6 I had gamescope-git 3.14.2  installed.   Now any attempt to remove upgrade of downgrade Gamescope cause the plasma 6 session to fail. 

If I reinstall back gamescope-git 3.14.2 then plasma Wayland work again.
Can Plasma and Gamescope share one files or something?

**Comment by VeilSilence** [comment]
Distro packaging problem.

**Comment by leucome** [comment]
> Distro packaging problem.

Yes, it was a packaging problem.  I forgot to tel, It was a chaotic aur package.

I went with a messy solution, without knowing what files was the culprits I manually extracted all files from the 3.14.2 package and put them back manually. It worked the Wayland session was able to start.  Then I installed the newer Gamescope and it seem to work as expected so far.  I might have orphan/left over files that cause issue later though.

I am suspecting  libdisplay-info vkroots files that were supplied in this package but I did not test further find the exact files.
